src/Sylius/Bundle/CoreBundle/Command/CancelUnpaidOrdersCommand.php

```diff
@@ -13,12 +13,41 @@
 
 namespace Sylius\Bundle\CoreBundle\Command;
 
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Doctrine\Common\Persistence\ObjectManager;
+use Sylius\Component\Core\Updater\UnpaidOrdersStateUpdaterInterface;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
-class CancelUnpaidOrdersCommand extends ContainerAwareCommand
+class CancelUnpaidOrdersCommand extends Command
 {
+    /**
+     * @var string
+     */
+    private $orderExpirationPeriod;
+
+    /**
+     * @var UnpaidOrdersStateUpdaterInterface
+     */
+    private $unpaidOrdersStateUpdater;
+
+    /**
+     * @var ObjectManager
+     */
```

Comment:
We could use short syntax in docblocks (`/** @var ObjectManager */`), here and above

Reply:
And in other files